.github/workflows/danger.yml

```
@@ -23,19 +27,30 @@ jobs:
           fetch-depth: 0
 
       - name: Download dangerfile.js
-        run: wget https://raw.githubusercontent.com/getsentry/github-workflows/${{ inputs._workflow_version }}/danger/dangerfile.js -P ${{ runner.temp }}
+        run: wget https://raw.githubusercontent.com/lucas-zimerman/sentry-github-workflows/refs/heads/lz/ext-danger/danger/dangerfile.js -P ${{ runner.temp }}
+
+      - name: Download external dangerfile (if provided)
+        if: inputs.extra-dangerfile
+        run: |
+          # Download the external dangerfile from the calling repository
+          wget "https://raw.githubusercontent.com/${{ github.repository }}/${{ github.head_ref || github.ref_name }}/${{ inputs.extra-dangerfile }}" -P ${{ runner.temp }}
+          echo "Downloaded external dangerfile: ${{ inputs.extra-dangerfile }}"
 
       # Using a pre-built docker image in GitHub container registry instaed of NPM to reduce possible attack vectors.
       - name: Run DangerJS
         id: danger
         run: |
           docker run \
+            --rm \
+            --interactive \
             --volume ${{ github.workspace }}:/github/workspace \
             --volume ${{ runner.temp }}:${{ runner.temp }} \
             --workdir /github/workspace \
-            --user $UID \
+            --user root \
             -e "INPUT_ARGS" -e "GITHUB_JOB" -e "GITHUB_REF" -e "GITHUB_SHA" -e "GITHUB_REPOSITORY" -e "GITHUB_REPOSITORY_OWNER" -e "GITHUB_RUN_ID" -e "GITHUB_RUN_NUMBER" -e "GITHUB_RETENTION_DAYS" -e "GITHUB_RUN_ATTEMPT" -e "GITHUB_ACTOR" -e "GITHUB_TRIGGERING_ACTOR" -e "GITHUB_WORKFLOW" -e "GITHUB_HEAD_REF" -e "GITHUB_BASE_REF" -e "GITHUB_EVENT_NAME" -e "GITHUB_SERVER_URL" -e "GITHUB_API_URL" -e "GITHUB_GRAPHQL_URL" -e "GITHUB_REF_NAME" -e "GITHUB_REF_PROTECTED" -e "GITHUB_REF_TYPE" -e "GITHUB_WORKSPACE" -e "GITHUB_ACTION" -e "GITHUB_EVENT_PATH" -e "GITHUB_ACTION_REPOSITORY" -e "GITHUB_ACTION_REF" -e "GITHUB_PATH" -e "GITHUB_ENV" -e "GITHUB_STEP_SUMMARY" -e "RUNNER_OS" -e "RUNNER_ARCH" -e "RUNNER_NAME" -e "RUNNER_TOOL_CACHE" -e "RUNNER_TEMP" -e "RUNNER_WORKSPACE" -e "ACTIONS_RUNTIME_URL" -e "ACTIONS_RUNTIME_TOKEN" -e "ACTIONS_CACHE_URL" -e GITHUB_ACTIONS=true -e CI=true \
             -e GITHUB_TOKEN="${{ github.token }}" \
             -e DANGER_DISABLE_TRANSPILATION="true" \
-            ghcr.io/danger/danger-js:11.3.1 \
-            --failOnErrors --dangerfile ${{ runner.temp }}/dangerfile.js
+            -e EXTRA_DANGERFILE_INPUT="${{ inputs.extra-dangerfile }}" \
+            -e RUNNER_TEMP="${{ runner.temp }}" \
+            ghcr.io/danger/danger-js:13.0.1 \
+            --verbose --failOnErrors --dangerfile ${{ runner.temp }}/dangerfile.js
```

lets remove the verbose flag.
```suggestion
            --failOnErrors --dangerfile ${{ runner.temp }}/dangerfile.js
```